tests/components/bond/test_cover.py

```diff
@@ -0,0 +1,113 @@
+"""Tests for the Bond cover device."""
+import logging
+
+from bond import BOND_DEVICE_TYPE_MOTORIZED_SHADES
+
+from homeassistant.components.bond.utils import BondDevice
+from homeassistant.components.cover import DOMAIN as COVER_DOMAIN
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    SERVICE_CLOSE_COVER,
+    SERVICE_OPEN_COVER,
+    SERVICE_STOP_COVER,
+)
+from homeassistant.helpers.entity_registry import EntityRegistry
+
+from .common import setup_platform
+
+from tests.async_mock import patch
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def test_entity_registry(hass):
+    """Tests that the devices are registered in the entity registry."""
+
+    with patch(
+        "homeassistant.components.bond.utils.get_bond_devices",
```

Comment:
The tests will be more robust if we patch the library instead of our code. Then we can refactor without changing tests.